### PR TITLE
OPSEXP-2880 Add audit-storage role

### PR DESCRIPTION
**Pull request by gionn**
OPSEXP-2880

### CODE REVIEW

**Review by pmacius — APPROVED** [review]
Are we disabling this role for older version?

**Comment by gionn** [comment]
> Are we disabling this role for older version?

good catch, let me add an explicit validation for that

**Review by pmacius — COMMENTED** [review]
We can bump dtas version to v1.6.0 and configure tests for audit-storage

**Review by pmacius — DISMISSED** [review]
LGTM

File: roles/audit_storage/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+audit_storage_artifact_name: alfresco-audit-storage-app
+
+audit_storage_username: alfresco
+audit_storage_group_name: alfresco
+
+audit_storage_server_port: 8083
+
+audit_storage_broker_url: failover:(nio://localhost:61616)?timeout=3000
+audit_storage_broker_username: admin
+audit_storage_broker_password: admin
```
Comment (alxgomz) [inline roles/audit_storage/defaults/main.yml:16]: why these default when we just don't set passwd for ES?

**Review by alxgomz — COMMENTED** [review]
Just a few comments, but maybe fix the password defaults in the module if there's not reason for them?

**Review by alxgomz — APPROVED** [review]

**Pull request merged**